.env-dist

```diff
@@ -1,6 +1,6 @@
 # https://nextjs.org/docs/messages/non-standard-node-env
 # development, production, test
-NODE_ENV=development
+NODE_ENV=production
```

Comment:
Wait... why this change? Does this affect what tests run? Ah, maybe https://github.com/mozilla/blurts-server/blob/c0aa2eb9cf0ce41b74d1527eadf8a78c26977569/.github/workflows/playwright.yml#L51

Reply:
Yes, that must be it! `NODE_ENV=development` is not a valid env variable when building Next.js with `npm run build`. This has been biting us in the past few days. Playwright only started failing on us after upgrading to`next: 13.5.4`. Building standalone without Playwright was still working.